src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using Native = Interop.CameraDeviceManager;
+
+namespace Tizen.Multimedia
+{
+    /// <summary>
+    /// This CameraDeviceManager class provides methods to controls current camera devices and gets its information.
```

Comment:
controls->control / gets->get (because of 'to')

Reply:
Done